nerfstudio/process_data/base_converter_to_nerfstudio_dataset.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2022 The Nerfstudio Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base class to processes a video or image sequence to a nerfstudio compatible dataset."""
+
+import logging
```

Comment:
Is there a reason you switched to logging from rich console? The rest of the codebase uses rich in order to get text effects, animations, emojis ect.

Reply:
I'm not super familiar with rich console so I might have missed it in the documentation when I quckly read it, so I'll be happy to revert that in case. I made that change here so:

1. Using logging enable the user to set different level of severity for the prints. If the user wants only Warning and Errors it's ok. If they wants info it's also ok. If they want nothing it's also fine :).
2. The user can log in a file instead of the terminal which can be useful for tracking the logs and sharing.

I use nerfstudio as a library to run algorithms in Azure ML studio for my company, and using the console prints makes it a bit harder to log specific type of error/warning/infos, or to simply bypass them when we don't need them. What do you think?

I just realized from this comment that some warning are logged as info which is wrong. I'll correct that too.

Reply:
I see, let's stick with rich for now. I think a larger change is probably needed across the codebase to accomplish what you are looking for. Here are two things that might be related - https://rich.readthedocs.io/en/stable/reference/console.html#rich.console.Console.capture
https://rich.readthedocs.io/en/stable/reference/logging.html

Reply:
Ok so I have made an update with the raise for the errors and using CONSOLE for "live logs". If you are against this, I can easily revert to using a `sys.exit` but I believe a raise is better for the user :). Up to you

Another advantage of using logging is that the code could be simplified by removing the use of a `summary_log`. Right now there is both a summary_log and a CONSOLE logging happening at the same time and we have two different process to log information and display them :). Using `logging` accross the board (to replace console and the summary_log gives more flexibility and control to the user for what is shown and when.

Let me know if it sounds interesting and I can submit another PR that does that for the IMage and Video dataset converter so you can have a look at what that would look like :).

Reply:
Fair enough, that's insight I didn't have!

What about adding a try...except in the `script/process_data.py` that will simply catch all raises and print the message that the contain? This should remove the stack trace and leave only the message, while giving programmer raises that they can handle :)?

Reply:
That sounds good to me.

Reply:
I'll add this tomorrow. It's late now ;). Thanks for the feedback

Reply:
Just a side note: you could have the best of both world with [rich console handler for logging](https://rich.readthedocs.io/en/stable/logging.html). Put logging everywhere, get rich console formatting ;)